CHANGELOG.md

```diff
@@ -8,6 +8,8 @@
 * add a Makefile target to import tea lambda cloudwatch group if getting an "The
 specified log group already exists" error: `make import-thin-egress-log`
 * add .gitconfig file to Docker image to mark /CIRRUS-core and /CIRRUS-DAAC as safe
+* Update Amazon linux to 2024
```

Comment:
"Update Amazon linux to 2024" confused because
Dockerfile line 1: `FROM amazonlinux:2023 as core_base`

Reply:
meant to put 2023. There is no 2024, Good catch.